README.md

```diff
@@ -1,41 +1,106 @@
-# Wall-E_v2.2-beta
+<!-- PROJECT LOGO -->
+[![Stargazers][stars-shield]][stars-url]
+[![Forks][forks-shield]][forks-url]
+[![Issues][issues-shield]][issues-url]
+[![License][license-shield]][license-url]
```

Comment:
Bring these badges below `Report Bug · Request Feature · Send a Pull Request`
Also align those in center